src/main/java/com/znsio/e2e/runner/Setup.java

```diff
@@ -90,19 +88,25 @@
     private static final String LAUNCH_NAME_SUFFIX = "LAUNCH_NAME_SUFFIX";
     private static final String APPIUM_UI_AUTOMATOR2_SERVER = "io.appium.uiautomator2.server";
     private static final String APPIUM_SETTINGS = "io.appium.settings";
-    private static final Logger LOGGER = Logger.getLogger(Setup.class.getName());
     public static final String WEBDRIVER_MANAGER_PROXY_URL = "WEBDRIVER_MANAGER_PROXY_URL";
+    private static final String REMOTE_WEBDRIVER_GRID_PORT_KEY = "REMOTE_WEBDRIVER_GRID_PORT_KEY";
+    public static final String BROWSER_CONFIG_FILE = "BROWSER_CONFIG_FILE";
```

Comment:
Do we need these to be public?

Reply:
pushed a change and made them package-protected